funnel/templates/email_account_reset.html.jinja2

```diff
@@ -0,0 +1,12 @@
+{% extends "email_layout.html.jinja2" %}
+{% block content -%}
+
+  <p>{% trans %}Hello {{ fullname }},{% endtrans %}</p>
```

Comment:
Does translation work in this case?
`{% trans name=user.fullname %}Hello {{ name }},{% endtrans %}`

Reply:
Both should work, but I'll check again.

Reply:
This does work. I'm merging this PR.